vi-start-to-finish.py

```diff
@@ -0,0 +1,223 @@
+"""Use dask-jobqueue to correct illumination, montage, and extract features."""
+import os
+import itertools
+import glob
+import sys
+import re
+import time
+import click
+import toolz as tz
+import numpy as np
+import pandas as pd
+from dask_jobqueue import SLURMCluster as Cluster
+from dask import delayed
+from dask.distributed import Client, as_completed
+from distributed.scheduler import KilledWorker
+from tqdm import tqdm
+from skimage import io
+from microscopium import preprocess as pre
+from microscopium._util import generate_spiral
+from microscopium import features as feat
+
+
+ROOT = '/scratch/su62/petermac/data'
```

Comment:
It'd be good if the user inputs were more clearly separated/called out, otherwise you feel like you have to check over every line of code before starting.

Reply:
Done